spec/unit/files/servicenow_spec.rb

```diff
@@ -30,8 +30,8 @@
 
   context 'node exists' do
     it "returns the node's parsed CMDB record" do
```

Comment:
@RandomNoun7 Do you agree w/ my rationale in the commit message re: this change?

Reply:
Sorry, but I don't think I do. The expected result of running the external node data script to query a servicenow instance is not to output the raw cmdb record. The data output by the script will be munged in a few different ways that make the expectation that the node data hash output by the script should match the cmdb record returned by servicenow not quite correct.

Reply:
We're thinking of "trusted external data" and "classification" as separate features. The "trusted external data" feature should output just the raw CMDB record. The only munging that the script does is for the "classification" bit and in that bit, we're already testing all of that munging.

This test is testing the "trusted external data" bit (when there is no "classification"). That bit is supposed to output the raw CMDB record.

Reply:
Ok, I think I finally see the different tests you're trying to run here. When I initially ran these acceptance tests with a realistic api response json as the mock this test fell apart and I think it confused me.